stl/inc/future

```diff
@@ -153,11 +153,9 @@ private:
 };
 
 // CLASS _Future_error_category

```

Comment:
Outdated comment.

Reply:
This is the kind of SHOUTY comment I don't like (it doesn't help in finding a pretty name) so will delete. (Same with the other ones)